tests/ui/associated-types/associated-type-shadowed-from-supertrait.stderr

```diff
@@ -0,0 +1,24 @@
+error[E0191]: the value of the associated type `X` in `Super` must be specified
+  --> $DIR/associated-type-shadowed-from-supertrait.rs:12:27
+   |
+LL |     type X;
+   |     ------ `Super::X` defined here
+...
+LL |     type X;
+   |     ------ `Super::X` shadowed here
+...
+LL | impl<T> Clone for Box<dyn Sub<X = T>> {
+   |                           ^^^^^^^^^^ associated type `X` must be specified
+   |
+help: consider renaming one of the associated types
```

Comment:
This help uses the same spans as the primary message above.
I suggest just extending the primary message labels with renaming suggestions in `is_local` cases, instead of adding a whole separate help.